client/src/components/JobInformation/CodeRow.vue

```diff
@@ -28,9 +28,9 @@ export default {
         return { expanded: false };
     },
     methods: {
-        toggleExpanded() {
+        expandCode() {
             if (this.codeItem) {
-                this.expanded = !this.expanded;
+                this.expanded = true;
```

Comment:
Why this change ? I think it's quite useful being able to toggle long outputs ?

Reply:
See https://github.com/galaxyproject/galaxy/pull/10980#issuecomment-7653500301 and @OlegZharkov's suggestion underneath.

Reply:
Can you come up with another solution ? It should be possible to use a combination of on:click and on:mousedown to prevent the toggle. Alternatively you could add an expand button that is dedicated to toggling the expanded state.